src/scripts/App.ts

```diff
@@ -11,8 +11,13 @@ class App {
         Preload.load(App.debug).then(function () {
             OakItemRunner.initialize();
             UndergroundItem.initialize();
-            App.game = new Game(new Breeding(), new Pokeballs(), new Wallet());
+            App.game = new Game(new Breeding(), new Pokeballs(), new Wallet(), new BadgeCase(16));
 
+            KeyItemHandler.initialize();
+            AchievementHandler.initialize();
+            player.gainKeyItem("Coin case", true);
+            player.gainKeyItem("Teachy tv", true);
+            player.gainKeyItem("Pokeball bag", true);
```

Comment:
These three lines seem more suited for KeyItemHandler.initialize()

Reply:
Agree, was more of a quickfix, I think they shouldn't exist anyway and just be part of the defaults in `KeyItems`. I'll fix it when refactoring KeyItems